Marlin/src/HAL/HAL_DUE/HAL_spi_Due.cpp

```diff
@@ -138,7 +87,7 @@
 
     /* The software SPI routine */
     __asm__ __volatile__(
-      ".syntax unified" "\n\t" // is to prevent CM0,CM1 non-unified syntax
+      A(".syntax unified") // is to prevent CM0,CM1 non-unified syntax
```

Comment:
If you're sure this works. The `A()` macro prepends a space, moving this directive to the second column. The original code has it in the first column.

Reply:
We could remove it althogether. This directive is for Cortex M0 MCUs, and none of the currently supported ones are 👍

Reply:
Well, if it has no ill side-effects, we should leave it in so we don't have to remember to put it back. Thinking ahead is good.